contrib/dracut/02zfsexpandknowledge/module-setup.sh.in

```diff
@@ -75,6 +93,14 @@ check() {
 # shellcheck disable=SC2154
 if [ -n "$hostonly" ]; then
 
+    if [ -f @sysconfdir@/default/zfs ]; then
```

Comment:
Nowhere else in the dracut config do we source `@sysconfdir@/default/zfs` for environment variables.  If we need to be able to specify this we should do it in a way which is consistent the the design described in `man/man7/dracut.zfs.7`.